pkg/devspace/build/build.go

```diff
@@ -106,10 +101,7 @@ func (c *controller) Build(ctx devspacecontext.Context, images []string, options
 
 			// Determine whether the local registry is required / enabled
 			isLocalReqistryRequired := !registry.HasPushPermission(imageConf)
-			useMinikubeDocker := registry.UseMinikubeDocker(ctx, imageConf)
-			if useMinikubeDocker {
-				ctx.Log().Warnf("Using Minikube for image %s, skipping local registry", imageConf.Image)
```

Comment:
We can bring back this edge case if desired, but it was somewhat unexpected to have `localRegistry.enabled = true` and it still not be used because of minikube.

This did require using a docker client configured for the minikube environment to work, see later comments.